### ISSUE TITLE
How would I transfer the VM to another computer?

### ISSUE THREAD

**Issue by kortistortoise** [myspaghetti/macos-virtualbox#505]
I have a working installation of Big Sur, but the question has been bothering me so much that I decided to create an account and ask: which folders/files would I need to transfer to another PC to get a working installation? Or, is the VM folder in C:\Users\Name\VirtualBox VMs enough to have a working installation? My main concern is with the NVRAM and EFI parameters. Is that bound to the installation? This is probably a stupid question, but I don't understand the script much further than the instructions. Thanks.

**Comment by myspaghetti** [comment]
* You can use VirtualBox's built-in VM export feature, but that tends to fail
* You can use the script to create a new virtual machine on the new host and configure it (`macos-guest-virtualbox.sh create_vm configure_vm`) and then attach the existing `.vdi` and overwrite the new `.nvram` file with the old `.nvram` file. The `.vdi` is where you created it with the script (or where you moved it) and the `.nvram` file is in the VirtualBox VM directory. Doing this will work assuming you didn't change the VirtualBox guest configuration in the old VM. The EFI parameters are set by the script/VirtualBox and I assume you didn't change them, and the `.nvram` variables are set by the script *and* the macOS guest and they do get changed (a bit) when you use macOS, but it's not a problem overwriting the `.nvram file`.

**TL;DR**: all you need is the `.vdi` file and the `.nvram` file and you can attach the `.vdi` to a new (configured) VM and overwrite the new `.nvram` with the old `.nvram`
